src/ontology/hp.Makefile

```diff
@@ -426,3 +426,20 @@ help:
 	@echo "$$data"
 	echo "Migrating EQs from MP to HP:"
 	echo "make ADOPT_EQS_MAPPING_URL=SOMEURL migrate_eqs_to_edit"
+
+.PHONY: diff
+diff: diff-table diff-kgcl diff-yaml
+
+tmp/hp-released.obo:
+	wget http://purl.obolibrary.org/obo/hp.obo -O $@ && cp ../../hp-full.obo tmp/hp-full.obo
+
+diff-table: tmp/hp-released.obo tmp/hp-full.obo
+	runoak -i simpleobo:tmp/hp-full.obo diff -X simpleobo:tmp/hp-released.obo \
+	-o difference_table.tsv --output-type csv --statistics --group-by-property oio:hasOBONamespace
+
+diff-kgcl: tmp/hp-released.obo tmp/hp-full.obo
+	runoak -i simpleobo:tmp/hp-full.obo diff -X simpleobo:tmp/hp-released.obo \
+	-o difference_kgcl.txt --output-type kgcl
```

Comment:
We need some output that looks something like:

https://github.com/monarch-initiative/mondo/releases/tag/v2022-11-01 (check the changelog). Nice tables which say:

- These terms were added
- these terms were obsoleted
- these labels where changed

And summary statistics similar to what you see in that Mondo issue.

Reply:
Should I write a script that does that separately or should this be implemented in `oaklib` itself. Thoughts?  

cc: @cmungall ?

Reply:
oaklib for sure. No scripts!